### PR TITLE
Change to use App Storage

### PR DESCRIPTION
**Pull request by hcarneiro**
- Tweaks for new App List - https://github.com/Fliplet/fliplet-widget-app-list/pull/40

### CODE REVIEW

**Comment by seromenho** [comment]
You can also remove `fliplet-security` from widget build dependencies.

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (squallstar) [inline .idea/workspace.xml]: ? what's this file?

Comment (seromenho) [reply .idea/workspace.xml]: ups

**Comment by seromenho** [comment]
@squallstar removed.

**Pull request merged**